src/main/java/core/basesyntax/service/fileservice/FileServiceImplForCsv.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.fileservice;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileServiceImplForCsv implements FileService {
```

Comment:
let's separate it in Reader and Writer

Reply:
I was told to not segregate it too much by one of the students (mankokolya), make these two methods in one class.

Reply:
done

Reply:
Support not to segregate too much. This service job is to work with a file it can do both operations read and write